modules/computer.c

```diff
@@ -563,15 +563,19 @@ gchar *callback_os(void)
         info_field(_("C Library"), computer->os->libc),
         info_field(_("Distribution"), computer->os->distro,
                    .icon = distro_icon),
+        computer->os->ubuntu_flavor
```

Comment:
`info_group_add()` returns a group pointer, so it's possible to add these optional fields without `info_field_skip()`:

```C
InfoGroup *version = info_add_group(info, _("Version"), ...,
                                    info_group_last());
if (computer->os->ubuntu_flavor) {
    info_group_add_fieldsv(group,
                          info_field(_("Spin/Flavor"),
                          computer->os->ubuntu_flavor->name,
                          .icon = computer->os->ubuntu_flavor->icon),
                          info_field_last());
}
```

Feel free to add a `info_group_add_field()` function that adds a single field to a group -- so you don't need `info_field_last()` here.

I'd like to avoid adding magic numbers and flags at this point; the idea is to clean up the code, removing anything not obvious.  The strings in GKeyFile are really difficult to understand already.

Reply:
Alright. New version uses info_group_add_field().